Sources/BoostBLEKit/HubProperty.swift

```diff
@@ -71,8 +135,48 @@ extension HubProperty: CustomStringConvertible {
             return "Button"
         case .firmwareVersion:
             return "Firmware Version"
+        case .hardwareVersion:
+            return "Hardware Version"
+        case .rssi:
+            return "RSSI"
         case .batteryVoltage:
             return "Battery Voltage [%]"
+        case .batteryType:
+            return "Battery Type"
+        case .manufacturerName:
+            return "Manufacturer Name"
+        case .radioFirmwareVersion:
+            return "Radio Firmware Version"
+        case .legoWirelessProtocolVersion:
+            return "LEGO Wireless Protocol Version"
+        case .systemTypeId:
+            return "System Type ID"
+        case .HWNetworkId:
+            return "H/W NetworkId"
+        case .primaryMacAddress:
+            return "Primary MAC Address"
+        case .secondaryMacAddress:
+            return "Secondary MAC Address"
+        case .hardwareNetworkFamily:
+            return "Hardware Network Family"
+        }
+    }
+    
+}
+
+extension HubProperty.Value: CustomStringConvertible {
+    
+    public var description: String {
+        switch self {
+        case .string(let string):
+            return string
+        case .bool(let bool):
+            return bool ? "yes" : "no"
```

Comment:
I prefer `bool.description` because `TRUE/FALSE` is used in the document.
https://lego.github.io/lego-ble-wireless-protocol-docs/index.html#hub-property-payload